packages/wonder-blocks-dropdown/components/action-item.test.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import renderer from "react-test-renderer";
+import {Link} from "react-router-dom";
+import {shallow} from "enzyme";
+import ReactRouterEnzymeContext from "react-router-enzyme-context";
+
+import ActionItem from "./action-item.js";
+
+describe("ActionItem", () => {
+    it("should render with disabled styles", () => {
+        const actionItem = renderer.create(
+            <ActionItem href="/foo" disabled={true} />,
+        );
+
+        expect(actionItem).toMatchSnapshot();
+    });
+
+    it("should render an anchor if there's no router", () => {
+        const wrapper = shallow(<ActionItem href="/foo" />);
+
+        expect(
+            wrapper
+                // call the props.children(state, handlers) with empty objects for both params
+                .renderProp("children")({}, {})
+                // shallow render the first child of that result to unwrap StyledAnchor HOC
+                .first()
+                .shallow()
```

Comment:
There's some discussion in https://github.com/airbnb/enzyme/issues/539 around possible API improvements to make this easier.